pages/linux/pct.md

```diff
@@ -0,0 +1,36 @@
+# pct
+
+> Manage LXC containers in Proxmox.
+> More information: <https://pve.proxmox.com/pve-docs/pct.1.html>.
+
+- List all containers:
+
+`pct list`
+
+- Start/Stop/Reboot a specific container:
+
+`pct {{start|stop|reboot}} {{100}}`
+
+- Access a specific container's shell:
+
+`pct enter {{100}}`
+
+- Create a container from template:
+
+`pct create {{100}} {{/var/lib/vz/template/cache/distro-name.tar.zst}} -hostname {{hostname}} -password {{password}} --rootfs {{local-lvm}} --on-boot`
+
+- Resize the container's disk to 20G:
+
+`pct resize {{100}} {{rootfs|mp0...mp255}} {{20G}}`
```

Comment:
Is `255` the limit here? In other disk-related commands we've used e.g. `sdX`.

Reply:
Hi @sbrl,

I am the editor of the line.
Not sure if there is a limit on mp254/mp255.
But, I agree with you the approach of mpX will be better.

💪